xgboost_ray/main.py

```diff
@@ -869,17 +914,20 @@ def train(params: Dict,
                 time.sleep(5)
                 queue.shutdown()
                 stop_event.shutdown()
+                #remove_placement_group(pg)
```

Comment:
Removing the placement group here causes the alive actors to die as well. I think we'd need to use placement group resizing here instead.

Reply:
Or I think another workaround we can do here is to disable the placement group-level fault tolerance.